_schemaV2.graphql

```diff
@@ -1292,6 +1292,9 @@ type Artwork implements Node & Searchable & Sellable {
   dimensions: dimensions
   displayLabel: String
   displayPriceRange: Boolean
+
+  # Domestic shipping fee.
+  domesticShippingFee: Money
```

Comment:
@starsirius - do you think we can rely on those to have real time artwork data that we can check on offer page?

Reply:
Yes I think so. When accessing from CMS, we'll use the authenticated loader which is not cached.

Reply:
Yes, I had to do [some digging](https://artsy.slack.com/archives/CP9P4KR35/p1614706681075900) to remind myself how loaders would work:
When auth token is present the auth loaders are merged into unauth ones and in case of `artworkLoader` where [auth'ed](https://github.com/artsy/metaphysics/blob/a85149e85316ea9252ba916432adf4cec297de93/src/lib/loaders/loaders_with_authentication/gravity.ts#L20) and [unauth'ed](https://github.com/artsy/metaphysics/blob/89543d6347e22c4365cb0e82c15a6c5c27290b9e/src/lib/loaders/loaders_without_authentication/gravity.ts#L51) have the same name the auth loader replaces the auth.